src/js/actions/export.js

```diff
@@ -149,6 +150,27 @@ define(function (require, exports) {
     };
 
     /**
+     * Get the port number of the generator plugin, as stored in the photoshop custom preferences
+     *
+     * @return {Promise.<?string>}
+     */
+    var _getWebSocketPort = function () {
+        var prefKey = ExportService.domainPrefKey;
+
+        return descriptor.playObject(preferenceLib.getCustomPreference(prefKey))
+            .then(function (pref) {
+                var settings = JSON.parse(objUtil.getPath(pref, prefKey + ".settings")),
+                    port = settings && settings.websocketServerPort;
+                log.debug("Export: Found configured port: " + port); // TEMP
```

Comment:
I find this useful when testing, especially when trying to run a separate external generator instance... but I assume *someone* will say this is too chatty